src/components/tests/Outline.js

```diff
@@ -59,4 +76,26 @@ describe("Outline", () => {
     await symbolsPromise;
     expect(component).toMatchSnapshot();
   });
+
+  it("should select a line of code in the current file on click", async () => {
+    const component = shallow(new OutlineComponent({ selectSource }));
+    const sourceId = "id";
+    const selectedSource = {
+      get: () => sourceId
+    };
+    const startLine = 12;
+
+    symbolDeclarations.functions[0] = {
```

Comment:
perhaps we also have a `generateSymbolDeclaration` factory as well so we can have the defaults in there